### PR TITLE
Upstream artifact for kata admission controller

### PR DESCRIPTION
**Pull request by krsna1729**
Get upstream artifacts of kata admission controller from
https://github.com/kata-containers/tests/tree/master/kata-webhook
Commit: 5ad2cec

Signed-off-by: Saikrishna Edupuganti <saikrishna.edupuganti@intel.com>

### CODE REVIEW

File: clr-k8s-examples/admit-kata/deploy/webhook.yaml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
```
Comment (grahamwhaley) [inline clr-k8s-examples/admit-kata/deploy/webhook.yaml]: As you have moved the path of the yaml files into the `deploy` subdir, do you also need to update the [README](https://github.com/clearlinux/cloud-native-setup/tree/master/clr-k8s-examples#making-kata-the-default-runtime) section so the `kubectl apply -f` points to the right place? afaict, `apply -f` is not recursive by default.

Comment (krsna1729) [reply clr-k8s-examples/admit-kata/deploy/webhook.yaml]: Thanks for the catch. Will probably remove most of the section in top level readme and put a pointer to this readme

**Comment by mcastelino** [comment]
@krsna1729 is this ready for merge?

**Pull request merged**